File: BEPilMoney/BEPilMoney/Models/Autenticacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BEPilMoney.Models
{
    public class Autenticacion
    {
        #region Propiedades
        public int Id { get; set; }
        public int IdUsuario { get; set; }
        public string Token { get; set; }
        public DateTime Fecha { get; set; }
        public int Estado { get; set; }
        #endregion
    }
}

File: Pil-Pr/src/app/models/Autenticacion.ts
export class Autenticacion{
    
    private Id?:number;
    private IdUsuario?:number;
    private Token?:string;
    private Fecha?:Date;
    private Estado?:number;

    constructor(id?:number, idUsuario?:number, token?:string, fecha?:Date, estado?:number){
        this.Id = id;
        this.IdUsuario = idUsuario;
        this.Token = token;
        this.Fecha = fecha;
        this.Estado = estado;
    }
    
}